Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+node("mobile-core-install-slave") {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage("Run Ansible scripts") {
+        def metadata_endpoint = "http://169.254.169.254/latest/meta-data"
```

Comment:
does this have to be hardcoded? do we want a follow up issue to remove it or will it never change

Reply:
Yes this should be hardcoded. Thats static variable